chpl/chpl-api/src/main/java/gov/healthit/chpl/questionableActivity/CertificationResultQuestionableActivityProvider.java

```diff
@@ -78,31 +99,39 @@ public QuestionableActivityCertificationResultDTO checkGapUpdated(
         }
         return activity;
     }
-    
+
+    /**
+     * Check for QA re: addition to G1 MACRA measures.
+     * @param origCertResult original certification criteria result
+     * @param newCertResult new certification criteria result
+     * @return list of added measures
+     */
     public List<QuestionableActivityCertificationResultDTO> checkG1MacraMeasuresAdded(
-            CertificationResult origCertResult, CertificationResult newCertResult) {
-        
-        List<QuestionableActivityCertificationResultDTO> addedMacras = new ArrayList<QuestionableActivityCertificationResultDTO>();
-        if ((origCertResult.getG1MacraMeasures() == null || origCertResult.getG1MacraMeasures().size() == 0) && 
-                newCertResult.getG1MacraMeasures() != null && newCertResult.getG1MacraMeasures().size() > 0) {
+            final CertificationResult origCertResult, final CertificationResult newCertResult) {
+
+        List<QuestionableActivityCertificationResultDTO> addedMacras
+        = new ArrayList<QuestionableActivityCertificationResultDTO>();
```

Comment:
Should this be indented?  https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent